packages/app/src/app/pages/common/Modals/PRModal/index.tsx

```diff
@@ -1,12 +1,16 @@
-import React from 'react';
-import { inject, observer } from 'app/componentConnectors';
 import { GitProgress } from 'app/components/GitProgress';
+import { useOvermind } from 'app/overmind';
+import React, { FunctionComponent } from 'react';
+
+export const PRModal: FunctionComponent = () => {
```

Comment:
There are files importing this that will fail

Reply:
Done @SaraVieira o/